docs/lessons/02-recap.md

```diff
@@ -0,0 +1,88 @@
+# 01 - Introduction to Intermediate Java
```

Comment:
This needs to be updated to something like `# 02 - Compiling a Java program` or something similar.

Reply:
🤦‍♂ copy paste :-)